src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.test.tsx

```diff
@@ -3,17 +3,55 @@ import { render } from '#app/components/react-testing-library-with-providers';
 import PortraitVideoPromo from '.';
 
 describe('PortraitVideoPromo', () => {
-  it('Should contain a h3 level title', () => {
+  it('Should contain a title', () => {
     const sampleHeadlines = { promoHeadline: 'Sample Heading' };
 
     const { container } = render(
       <PortraitVideoPromo id="testId" headlines={sampleHeadlines} />,
     );
 
-    const heading = container.querySelector('h3');
+    const heading = container.querySelector('p')?.querySelectorAll('span')[1];
     expect(heading?.innerHTML).toBe('Sample Heading');
   });
 
+  it('Should contain a visually hidden text with required screen reader details for the component - with duration', () => {
+    const sampleHeadlines = {
+      promoHeadline: 'Sample Heading',
+    };
+    const sampleVideoData = {
+      id: 'p01wjx7v',
+      version: {
+        duration: 'PT13S',
+        kind: 'programme',
+        territories: ['uk', 'nonuk'],
+      },
+    };
+
+    const { container } = render(
+      <PortraitVideoPromo
+        id="testId"
+        headlines={sampleHeadlines}
+        video={sampleVideoData}
+      />,
+    );
+
+    const heading = container.querySelector('p')?.querySelectorAll('span')[0];
+    expect(heading?.innerHTML).toBe(
+      'Sample Heading, video, Duration  0:13, Play video',
```

Comment:
Looks like there's an extra space here - can we remove?
```suggestion
      'Sample Heading, video, Duration 0:13, Play video',
```